src/journal/src/Journal/Internal/ByteBuffer.hs

```diff
@@ -216,11 +239,17 @@ getByteAt = undefined
 ------------------------------------------------------------------------
 -- * Multi-byte operations
 
-putBytes :: ByteBuffer -> [Word8] -> IO ()
-putBytes = undefined
+putBytes :: ByteBuffer -> ByteBuffer -> IO ()
+putBytes src dest = do
```

Comment:
Should this function update the position for the destination?

Reply:
Fixed, thanks.